workos/sso.py

```diff
@@ -213,13 +223,11 @@ def list_connections(
                     "Passing a string value as the 'connection_type' parameter for 'list_connections' is deprecated and will be removed in the next major version. Please pass a 'ConnectionType' instead.",
                     DeprecationWarning,
                 )
-            except KeyError:
-                raise ValueError("'connection_type' must be a member of ConnectionType")
+            except KeyError as key_error:
+                raise ValueError("'connection_type' must be a member of ConnectionType") from key_error
```

Comment:
CI is failing due this this being considered invalid syntax in Python 2.7:

```
./workos/sso.py:227:93: E999 SyntaxError: invalid syntax
                raise ValueError("'connection_type' must be a member of ConnectionType") from key_error
```

We haven't yet dropped supported for Python 2 in the WorkOS SDK, so we'll need to maintain compatibility for the time-being.

Reply:
Thanks @maxdeviant, updated! 

Happy to add pre-commit hooks with black etc in a separate PR so that files are correctly formatted on committing changes before CI checks and Tox for running tests across the different supported python versions locally if you'd like.

Reply:
> Happy to add pre-commit hooks with black etc in a separate PR

Thanks for the offer, but we're not interested in adding pre-commit hooks to the repo at this time.

Reply:
All good, thanks.